### PR TITLE
Package satex.0.1.1

### PR DESCRIPTION
**Pull request by smimram**
### `satex.0.1.1`
String diagram generator for LaTeX
Satex takes descriptions of string diagrams in the form of categorical compositions and converts them to string diagrams drawn in TikZ.

---
* Homepage: https://github.com/smimram/satex
* Source repo: git+https://github.com/smimram/satex.git
* Bug tracker: https://github.com/smimram/satex/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/satex/satex.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+synopsis: "String diagram generator for LaTeX"
+description:
+  "Satex takes descriptions of string diagrams in the form of categorical compositions and converts them to string diagrams drawn in TikZ."
+maintainer: ["Samuel Mimram <smimram@gmail.com>"]
+authors: ["Samuel Mimram <smimram@gmail.com>"]
+license: "GPL-3.0"
+homepage: "https://github.com/smimram/satex"
+bug-reports: "https://github.com/smimram/satex/issues"
+depends: [
+  "dune" {>= "2.0"}
```
Comment (avsm) [inline packages/satex/satex.0.1.1/opam:12]:
```suggestion
  "dune" {>= "2.0"}
  "ocaml" {>="4.07.0"}
```

for Float module

File: packages/satex/satex.0.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "menhir" {build}
```
Comment (avsm) [inline packages/satex/satex.0.1.1/opam:13]:
```suggestion
  "menhir" {>="20180528"}
```

for infer-write-query use.  You also need to be careful of {build} here -- if Menhir is recompiled and generates a new automata, then you will need to rebuild this package. I'll remove the build tag from the published opam release.

File: packages/satex/satex.0.1.1/opam [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "Satex takes descriptions of string diagrams in the form of categorical compositions and converts them to string diagrams drawn in TikZ."
+maintainer: ["Samuel Mimram <smimram@gmail.com>"]
+authors: ["Samuel Mimram <smimram@gmail.com>"]
+license: "GPL-3.0"
+homepage: "https://github.com/smimram/satex"
+bug-reports: "https://github.com/smimram/satex/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>="4.07.0"}
+  "menhir" {>="20180528"}
```
Comment (mseri) [inline packages/satex/satex.0.1.1/opam:14]:
```suggestion
  "menhir" {>="20180703"}
```
Uses `$loc`, introduced there

File: packages/satex/satex.0.1.1/opam [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/satex/satex.0.1.1/opam:17]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/satex/satex.0.1.1/opam [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "String diagram generator for LaTeX"
+description:
+  "Satex takes descriptions of string diagrams in the form of categorical compositions and converts them to string diagrams drawn in TikZ."
+maintainer: ["Samuel Mimram <smimram@gmail.com>"]
+authors: ["Samuel Mimram <smimram@gmail.com>"]
+license: "GPL-3.0"
```
Comment (mseri) [inline packages/satex/satex.0.1.1/opam:8]:
```suggestion
license: "GPL-3.0-or-later"
```
According to the LICENSE file in the repo

**Comment by smimram** [comment]
Thanks for your changes, I have reported them upstream so that they will also be applied for future versions.

**Comment by mseri** [comment]
Looks good now, thanks!

**Pull request merged**